tests/bluetooth/tester/src/audio/btp_bap_broadcast.c

```diff
@@ -484,13 +489,24 @@ uint8_t btp_bap_broadcast_source_stop(const void *cmd, uint16_t cmd_len,
 
 	LOG_DBG("");
 
+	k_sem_reset(&sem_stream_stopped);
+
 	err = bt_bap_broadcast_source_stop(source->bap_broadcast);
 	if (err != 0) {
 		LOG_DBG("Unable to stop broadcast source: %d", err);
 
 		return BTP_STATUS_FAILED;
 	}
 
+	for (int i = 0; i < source->stream_count; i++) {
+		err = k_sem_take(&sem_stream_stopped, K_MSEC(1000));
```

Comment:
to be very strict that timeout should be for all semaphores,  but since this will indicate bug in stack I don't think it matter much and adding time tracking would be overkill :-)